base_user_role_history/views/base_user_role_line_history.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2019 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <!--FORM view-->
+    <record model="ir.ui.view" id="base_user_role_line_history_form_view">
+        <field
+            name="name"
+        >base.user.role.line.history.form (in base_user_role_history)</field>
+        <field name="model">base.user.role.line.history</field>
+        <field name="arch" type="xml">
+            <form create="0" edit="0" delete="0">
+                <header>
+                </header>
```

Comment:
Not used, to remove? (or maybe you override this section in a custom module?)
```suggestion
```

Reply:
I don't know the user of this <header> but I prefer to keep it as it could be useful for xpath. 
This PR purpose is to migrate this module without breaking something else. If you want you can create a branch afterward and suggest an improvement after analyzing the modules that depends on this one.
Thanks

Reply:
All good, didn't saw it was already there in 13.0.